CompilerSource/compiler/compile.cpp

```diff
@@ -725,6 +726,7 @@ int lang_CPP::compile(const GameData &game, const char* exe_filename, int mode)
   make += "NETWORKING=\""  + extensions::targetAPI.networkSys + "\" ";
   make += "PLATFORM=\"" + extensions::targetAPI.windowSys + "\" ";
   make += "TARGET-PLATFORM=\"" + compilerInfo.target_platform + "\" ";
+  make += "-j" + num_make_jobs + " ";
```

Comment:
This will default the number of jobs to unbounded, which is likely to cause problems on some systems.

Reply:
Check previous comment, default value is 1. I tested without `-j` and it worked fine, using only 1 job

Reply:
emake sends a default but lgm does not have one

Reply:
Personally I'd just default to empty and predicate this line (and 367) on `!num_make_jobs.empty()`. But I guess the default of "1" in `parse_ide_settings` is good enough.